### DISCUSSION TITLE
Supabase SQL editor is showing me the current user as postgres

### DISCUSSION THREAD

**Discussion by ashishkankal** [supabase/supabase#12101] [Questions]
Supabase SQL editor is showing the current user as `postgres`. However it should show as `supabase_admin`. which it was showing earlier. How to change the current user `supabase_admin` ?

<img width="572" alt="image" src="https://user-images.githubusercontent.com/7954909/216032688-b14ae653-6a95-40e5-9656-5235485249a3.png">

**Comment by GaryAustin1** [comment]
Supabase has removed super user privilege from the dashboard and changed the user to postgres (without super user privilege).
 https://github.com/supabase/supabase/discussions/9314

**Reply by Speaka** [reply]
I have the same issue as mentioned. Due to this i'cant access tables in schemas which were created as supabase_admin a week ago. This should not have been possible. How to solve this? 
By the way: if you create a new empty project the user is still supabase_admin not postgres.
